apprentice_teacher.rb

```diff
@@ -9,10 +16,6 @@ def initialize(options={})
     @phase = 3
```

Comment:
The phase was set the same for both types of teachers.  You should pull those up.